detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/config/ValidateConfig.kt

```diff
@@ -140,3 +142,12 @@ internal fun propertyIsDeprecated(
         "Property '$prop' is deprecated. $deprecationDescription.",
         if (reportAsError) Notification.Level.Error else Notification.Level.Warning,
     )
+
+internal fun propertyShouldBeAnArray(
+    prop: String,
+    reportAsError: Boolean,
+): Notification =
+    SimpleNotification(
+        "Property '$prop' should be an array instead of a String.",
```

Comment:
```suggestion
        "Property '$prop' should be a YAML array instead of a comma-separated String.",
```

I know this is a bit risky, but it's more clear from the user prespective